module/zfs/brt.c

```diff
@@ -1734,4 +1733,8 @@ ZFS_MODULE_PARAM(zfs_brt, zfs_brt_, prefetch, INT, ZMOD_RW,
 #ifdef ZFS_BRT_DEBUG
 ZFS_MODULE_PARAM(zfs_brt, zfs_brt_, debug, INT, ZMOD_RW, "BRT debug");
 #endif
+ZFS_MODULE_PARAM(zfs_brt, , brt_zap_default_bs, UINT, ZMOD_RW,
+	"BRT ZAP leaf blockshift");
+ZFS_MODULE_PARAM(zfs_brt, , brt_zap_default_ibs, UINT, ZMOD_RW,
+	"BRT ZAP indirect blockshift");
```

Comment:
These names should really use the same prefixes as the `prefetch` and `debug` entries above.  It looks like we actually never documented the prefetch and debug tunables so we could modify those names a bit if needed without too much disruption.